dashboard-frontend/src/pages/AccountsPage/AccountList.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react';
+import { AccountResource } from '../../types/account';
+import PrimaryButton from '../../components/PrimaryButton';
+import InputTypeformKey from './InputTypeformKey';
+import InputFlyAPIKey from './InputFlyAPIKey';
+import useCreateAccount from './useCreateAccount';
+
+const AccountList: React.FC<any> = ({
```

Comment:
Im not sure I understand :thinking: I thought in typescript you should type everything so that the compiler knows the types its dealing with? Although I might be wrong

Reply:
That's my understanding too tbh! let's reverse what I said, probably the right thing to do is to add `React.FC` to all other components rather than leave it out here